src/styles/typography/index.md.njk

```diff
@@ -118,14 +118,23 @@ If you're displaying lots of links together and want to save space and avoid rep
   - adding a line of text before the links saying 'The following links open in a new tab'
   - including `<span class="govuk-visually-hidden">(opens in new tab)</span>` as part of the link text, so that part of the link text is visually hidden but still accessible to screen readers
 
+## Links on dark backgrounds
+
+Use the `govuk-link--inverse` modifier class to show white links on dark backgrounds - for example, in headers, custom components, and patterns with darker backgrounds.
+
+Make sure all users can see the links - the white links and background colour must have a contrast ratio of at least 4.5:1.
+
+**[Example - govuk-link--inverse]**
+
 ### Links without underlines
 
-Only remove underlines from links if:
+Use the `govuk-link--no-underline` modifier class to remove underlines from links.
+
+Only do this if the context is clear to the user that they’re links, even without the underline. 
```

Comment:
By "they're," we're referring to the links - so is that like saying we're making it clear to the user that the links are in fact links?

What about this wording?
"Only do this if the context tells the user that the text is a link, even without the underline."

Reply:
Yeah, sounds good to me!
```suggestion
Only do this if the context tells the user that the text is a link, even without the underline. 
```